assets/src/edit-story/components/keyboardShortcutsMenu/karma/keyboardShortcuts.karma.js

```diff
@@ -0,0 +1,230 @@
+        'keyboard-shortcuts-toggle'
+      );
+
+      await fixture.events.click(menuToggle);
+
+      const openMenu = await fixture.screen.queryByTestId(
+        'keyboard-shortcuts-menu'
+      );
+
+      expect(openMenu).toBeTruthy();
+    });
+
+    describe('when menu is open', () => {
+      beforeEach(async () => {
+        const menuToggle = fixture.screen.getByTestId(
+          'keyboard-shortcuts-toggle'
+        );
+
+        await fixture.events.click(menuToggle);
+      });
+
+      it('should be able to close menu by clicking on close button', async () => {
+        const openMenu = await fixture.screen.getByTestId(
+          'keyboard-shortcuts-menu'
+        );
+        expect(openMenu).toBeTruthy();
+
+        const closeButton = fixture.screen.queryByRole('button', {
+          name: /^Close menu$/,
+        });
+        await fixture.events.click(closeButton);
+        // Give time for menu to close
+        await fixture.events.sleep(100);
+
+        const closedMenu = await fixture.screen.queryByTestId(
+          'keyboard-shortcuts-menu'
+        );
+
+        expect(closedMenu).toBeNull();
+      });
+
+      it('should be able to close menu by clicking outside the menu', async () => {
+        const openMenu = await fixture.screen.getByTestId(
+          'keyboard-shortcuts-menu'
+        );
+        expect(openMenu).toBeTruthy();
+
+        // Click outside menu
+        await fixture.events.mouse.clickOn(openMenu, -10, -10);
+        // Give time for menu to close
+        await fixture.events.sleep(100);
+
+        const closedMenu = await fixture.screen.queryByTestId(
+          'keyboard-shortcuts-menu'
+        );
+
+        expect(closedMenu).toBeNull();
+      });
+
+      it('should not close menu when clicking inside the menu', async () => {
+        const openMenu = await fixture.screen.getByTestId(
+          'keyboard-shortcuts-menu'
+        );
+        expect(openMenu).toBeTruthy();
+
+        // Click inside menu
+        await fixture.events.mouse.clickOn(openMenu, 10, 10);
+
+        const menuItem = await within(openMenu).queryAllByRole('listitem')[0];
+        // Click on element inside menu
+        await fixture.events.click(menuItem);
+        // Give time for menu to process clicks
+        await fixture.events.sleep(100);
+
+        const menu = await fixture.screen.queryByTestId(
+          'keyboard-shortcuts-menu'
+        );
+
+        // Menu should still be open
+        expect(menu).toBeTruthy();
+      });
+    });
+  });
+
+  describe('CUJ: User can interact with menu using keyboard: Tab to menu, enter to open, esc to close', () => {
+    it('should be able to tab to keyboard shortcuts button', async () => {
+      const safeZone = fixture.screen.getByTestId('safe-zone-toggle');
+
+      await fixture.events.focus(safeZone);
+
+      await fixture.events.keyboard.down('Shift');
+      await fixture.events.keyboard.press('Tab');
+      await fixture.events.keyboard.up('Shift');
```

Comment:
You can just use `await fixture.events.keyboard.shortcut('Shift+Tab')` here - it's a shortcut for this sequence.